src/delta_lake/delta_service.py

```diff
@@ -33,6 +35,57 @@
     "vacuum",
 }
 
+DISALLOW_SQL_META_CHARS = {
+    "--",
+    "/*",
+    "*/",
+    ";",
+    "\\",
```

Comment:
Why is double backslash disallowed?

Reply:
it was suggested in the Ticket:https://kbase-jira.atlassian.net/browse/CDM-120

I actually think only ";" matters here but also is checked by `if len(statements) != 1:`.

Reply:
I asked in the ticket. There are some legit use cases: https://medium.com/@ichsanputr/be-careful-with-backslash-in-mysql-database-0532b9de6a36

That said, easy enough to remove later